flytekit/types/structured/basic_dfs.py

```diff
@@ -55,14 +55,13 @@ def decode(
         self,
         ctx: FlyteContext,
         flyte_value: literals.StructuredDataset,
+        current_task_metadata: typing.Optional[StructuredDatasetMetadata] = None,
```

Comment:
can we make this not optional?  we can always make one right?  Or am i missing something?

Reply:
you're right, `StructuredDatasetMetadata` will never be None, thanks